mlir/include/mlir/Dialect/Tosa/IR/TosaUtilOps.td

```diff
@@ -115,11 +116,11 @@ def Tosa_VariableWriteOp : Tosa_Op<"variable.write", []> {
   let summary = "write_buffer operator";
 
   let description = [{
-    Assigns a value to pseudo-buffer resource holding a mutable tensor.
+    Assigns a value to the pseudo-buffer resource holding a persistent mutable tensor.
   }];
 
   let arguments = (ins
-    SymbolNameAttr:$name,
+    I32Attr:$uid,
     AnyType:$value
```

Comment:
Should this be `TosaTensor: $input1` in accordance with the spec?

Reply:
the problem is: `TosaTensor` is an input type while we need an attribute here.

Reply:
Updated to align with spec.